frontend/src/fixtures/recommendationRequestsFixtures.js

```diff
@@ -0,0 +1,56 @@
+const recommendationRequestsFixtures = {
+  oneRecommendationRequest: [
+    {
+      id: 1,
+      requesterEmail: "ramonwang@ucsb.edu",
+      requesterName: "Ramon Wang",
+      professorEmail: "pconrad@ucsb.edu",
+      professorName: "Phil Conrad",
+      requestType: "CS Department BS/MS program",
```

Comment:
Ultimately, these might be better as foreign keys into the users table (for requester and professor) and for requestType.  But for now, this is fine.

Reply:
Okay 👍  we'll use foreign keys once those are set up.